stubs/openpyxl/openpyxl/xml/functions.pyi

```diff
@@ -1,6 +1,74 @@
+import sys
```

Comment:
This file... is a mess...

The lack of import conditionals in type checkers, and/or `AnyOf` really hurts here.

Pyright handles it a bit better than mypy (as per the comment below)

I could extract all functions and manually overload them.

But I still have one big issue with `Element`, which is either a Class (note: a class, not an instance) or a function. Depending on where it's imported from.